.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+    - "3.8"
+
+install: pip install -r requirements.txt
```

Comment:
The "install" section should include instructions for installing the application using setup.py. Dependencies should be installed as part of this process, without explicitly using pip.

Reply:
done